src/sidebar/helpers/thread-sorters.ts

```diff
@@ -66,65 +66,75 @@ function compareStrings(a: string, b: string): number {
   }
 }
 
-type SortFunction = (a: Thread, b: Thread) => number;
+export type Options = {
+  sortBy: 'newest' | 'oldest' | 'location';
```

Comment:
It might make sense in future to split this into separate fields for what to sort by and whether to use ascending or descending order. I haven't done that yet as it would create new sort possibilities (sort by location reversed) which we don't need.